cishouseholds/impute.py

```diff
@@ -10,6 +10,63 @@
 from pyspark.sql.types import DoubleType
 from pyspark.sql.window import Window
 
+from cishouseholds.edit import update_column_values_from_map
+
+
+def impute_think_had_covid(
+    df: DataFrame,
+    participant_id_column: str,
+    date_column: str,
+    visit_date_column: str,
+    type_column: str,
+    contact_column: str,
+    hierarchy_map: dict,
+):
+    """
+    impute think had covid and associated variables
+    Parameters
```

Comment:
Don't worry about adding these to the docstring unless you need to provide additional description of each parameters, as the names can be seen in the `def` above